telethon/tl/custom/participantpermissions.py

```diff
@@ -0,0 +1,29 @@
+from .. import types
+
+
+class ParticipantPermissions:
+    """
+    Participant permissions information
+    """
+    def __init__(self, participant):
+        self.participant = participant
+
+    @property
+    def is_admin(self):
+        return self.is_creator or isinstance(self.participant, types.ChannelParticipantAdmin)
+
+    @property
+    def is_creator(self):
+        return isinstance(self.participant, types.ChannelParticipantCreator)
+
+    @property
+    def is_default_permissions(self):
```

Comment:
All of these properties need a description, and perhaps better naming. For instance, `has_default_permissions` would be a bit more fitting. I believe this also should be true when the participant is self? Though I haven't tested.